ch02/probability_intro.md

```diff
@@ -0,0 +1,51 @@
+
+When two events $A$ and $B$ aren't disjoint (they share common outcomes), then the probability that $A$ or $B$ ($A \cup B$) occur is the sum of the probability of each event, minus the probability of both events occurring ($A \cap B$). $P(A \cup B) = P(A) + P(B) - P(A \cap B) \text{ if } A \cap B \neq \varnothing$. The intuition behind this can better be visualized through a venn diagram.
+
+![probability_intro_venn](../assets/probability_intro_venn.png)
+
+When we have two events $A$ and $B$ that have common outcomes and we want to find the probability that either event happens, we want the area inside the circles. However, if we add together the probabilities of $A$ occurring and $B$ occurring, we count the intersection of the circles, $ P(A \cap B) $, twice. That's why we need so subtract the intersection once.
+
+For example, let's say we wish to find the probability of the events "getting two heads" or "getting at least 1 head" in two coin tosses. We can see that there are 3 outcomes in our sample space that have at least one head, and 1 outcome that has two heads. If we were to add these probabilites together, we would have $ \frac{3}{4} + \frac{1}{4} = 1$ ! Clearly, this result is wrong because we could get two tails, which isn't an outcome in either event. We did not account for the fact that the two events share an outcome; getting $ \{HH\} $ falls under both events, so we need to subtract the probability of that occuring. The correct probability of both events occuring would be $ \frac{3}{4} + \frac{1}{4} - \frac{1}{4} = \frac{3}{4} $
+
+## Multiplication Rule
+
+Other times, we wish to calculate probabilities involving one event AND another event occurring. To do so, we multiply their probabilities.
+
+In some cases, the first event happening does not impact the probability of the second event happening. We call these **independent events**. Suppose we want to calculated the probability of getting heads on a fair coin toss and getting a six on a fair dice roll. We can take into account that getting heads on a coin toss doesn't have any effect on the result of rolling the dice together. $ \frac{1}{2} $ the time we will get heads, and in that fraction of the time, we will only get a six $ \frac{1}{6} $ of the time. To find a fraction of a fraction, we can multiply them together. $P(\text{getting heads and a 6}) = \frac{1}{2} \* \frac{1}{6} = \frac{1}{12}$. In general for independent events *A* and *B*, $P(A \text{ and } B) = P(A)\*P(B)$
```

Comment:
First sentence: "the occurrence of the first event does not impact the probability of the second"
Second sentence: "calculate" instead of "calculated"
Second sentence: Maybe put quotes around the two events for clarity.
Third sentence: "[...] the result of the dice roll."
Fourth sentence: Take out the in "in that fraction of the time"
Last sentence: I would use "A \cap B" instead of "A \text{and} B" to make the expression more formulaic.
Last sentence: Put a period at the end.

Reply:
fixed